### PR TITLE
ES-96/SBOM generation

### PR DESCRIPTION
**Pull request by seanbrereton**
Creating SBOM using the cyclonedx plugin.
Adding a task to zip the sboms into 1 folder and publish them.
Tested locally: https://gradle.dev.r3.com/s/zomuydafgew2e

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — COMMENTED** [review]
sorry I missed the typo ;)

File: settings.gradle [hunk 1]
```diff
@@ -40,6 +40,7 @@ pluginManagement {
         id 'com.github.ben-manes.versions' version dependencyCheckVersion
         id "com.jfrog.artifactory" version artifactoryPluginVersion
         id 'io.snyk.gradle.plugin.snykplugin' version snykVersion
+        id "org.cyclonedx.bom" version "1.7.4"
```
Comment (ronanbrowne) [inline settings.gradle]: Can we externalize this property to gradle.properties file in line with others above this

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
minor comment re version

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**